README.md

```diff
@@ -140,7 +140,7 @@ The following settings are stored in `template_config.yml`.
   pydocstyle            Boolean, whether to have flake8 use pydocstyle to check for compliance with
                         Python docstring conventions.
 
-  python_version        Python version to use in the CI. Currently only 3.6 and 3.8 are supported.
+  python_version        Python version to use in the CI.
```

Comment:
```suggestion
  python_version        Python version to use in the CI. This does not affect the python version inside the CI container.
```

Actually, i hope, we can choose different base images in the test scenarios. -> Separate PR